marketpulse/main/views.py

```diff
@@ -120,7 +120,7 @@ def list_contributions(request, user=None):
 
     contributions = Contribution.objects.filter(activity__name=FFXOS_ACTIVITY_NAME)
     if user:
-        contributions = Contribution.objects.filter(user=user, activity__name=FFXOS_ACTIVITY_NAME)
+        contributions = contributions.filter(user=user)
```

Comment:
Probably you need to remove this commit completely. With the gallery coming in a different url, we need to filter only the FirefoxOS activities.